commons/src/config/log_id.rs

```diff
@@ -20,6 +20,6 @@ pub enum ConfigErr {
     #[error("Given locale is not in default locales data. Please provide data or use one of supported default locales: en-US, de-AT, bs-BA")]
     BadLocaleUsed,
     /// Log-id denoting provided locales are not included in default locales data
-    #[error("Given locale is missing keys in the used data file. Falling back to default locale for the given language.")]
-    LocaleMissingKeys,
+    #[error("Given locale is incompatible with data file. Using falling locale '{}'.", .0)]
```

Comment:
This error is now not displayed at all, since it mostly does not have any impact on users. Locale is not relevant except for splitting graphemes, and that should work the same with "fallback" locale used. Therefore, this is now logged as debug information.